ReleaseNotes.md

```diff
@@ -1,3 +1,5 @@
 #Release Notes
 
-- Update SDK 15.4.1
\ No newline at end of file
+- Update SDK 15.5.1
```

Comment:
Could you also mention Update Target Framework to Android 12?, for SDK release notes too.

Reply:
Sure